packages/react/src/utils/types.ts

```diff
@@ -0,0 +1,21 @@
+import type { InboxNotification } from '@novu/js';
+import type { BaseNovuUIOptions } from '@novu/js/ui';
+
+export type Notification = InboxNotification;
```

Comment:
☑ **chore:** We shouldn't use the `InboxNotification` for the public interface, please update it to the `Notification` class. Please check the other places too.